src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -65,7 +71,7 @@ const ListFilters: React.FC<{
 
   const resetFilters = () => {
     updateQuery(Object.fromEntries(activeFilters.map((filterName) => [filterName, undefined])))
-    setActiveFilters([])
+    setActiveFilters(defaultDisplayFilters)
```

Comment:
Thanks for fixing this. 

Just one tiny nit -- can you change the string for the label from "Clear all filters" to "Reset all filters", as "clear" doesn't quite make sense now if we're leaving the default ones up.